lib/features/BentaSignUp/SignUpPage2.dart

```diff
@@ -0,0 +1,216 @@
+import 'package:flutter/material.dart';
+import 'package:cc206_benta/features/BentaSignUp/SignUpPage1.dart';
+import 'package:cc206_benta/features/BentaSignUp/CreatingAccountPage.dart';
+
+
+class SignUpPage2 extends StatefulWidget{
+  const SignUpPage2({ Key? key }) : super(key: key);
+
+  @override
+  _SignUpStep2 createState() => _SignUpStep2();
+}
+
+
+class _SignUpStep2 extends State<SignUpPage2> {
+
+  final _formKey = GlobalKey<FormState>();
+
+  bool _isVisible = false;
+  bool _isVisible2 = false;
+
+  String? _password;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color(0xFF579008),
```

Comment:
So that's how you use hex values in Flutter for colors... I use RGB instead on my code.